frontend/apps/marketing/src/components/iframe/iframeContentfulDefinition.ts

```diff
@@ -0,0 +1,48 @@
+// Creates a definition for the Iframe component to be used in Contentful Studio
+import {ComponentDefinition} from '@contentful/experiences-sdk-react';
+
+export const IframeContentfulComponentDefinition: ComponentDefinition = {
+  id: 'iframe',
+  name: 'iFrame',
+  category: 'Advanced',
+  thumbnailUrl:
+    'https://images.ctfassets.net/90t6bu6vlf76/1qy9FC9Bqb4ADrpyszIa5M/5e791afb110fc5bc8caa95710c1dea16/component_code_thumbnail.png',
+  tooltip: {
+    description:
+      'Embed external content using an iframe. Ideal for embedding videos, interactive tools, or third-party widgets within a page.',
+    imageUrl:
+      'https://images.ctfassets.net/90t6bu6vlf76/75ulYKJrhP83vfIre5Rm88/8e94a8b30dc113217b6399f19ebaf3c1/component_code_tooltip.png',
```

Comment:
Super nit: rename this image file in Contentful (and here) to be consistent
```suggestion
      'https://images.ctfassets.net/90t6bu6vlf76/75ulYKJrhP83vfIre5Rm88/8e94a8b30dc113217b6399f19ebaf3c1/component_iframe_tooltip.png',
```